framework/src/base/FEProblem.C

```diff
@@ -2217,8 +2206,53 @@ FEProblem::computeIndicatorsAndMarkers()
 }
 
 void
-FEProblem::computeUserObjectsInternal(ExecFlagType type, UserObjectWarehouse::GROUP group)
+FEProblem::execute(const ExecFlagType & exec_type)
 {
+  // Set FEProblem execution flag
+  if (exec_type == EXEC_TIMESTEP_BEGIN)
+    onTimestepBegin();
+
+  else if (exec_type == EXEC_TIMESTEP_END)
+    onTimestepEnd();
+
+  // Pre-aux UserObjects
+  Moose::perf_log.push("computeUserObjects()", "Solve");
```

Comment:
Since you are moving perf_log stuff around, I'll point out that this isn't quite right (never was). This only times the computation of user objects outside of the solve (e.g. doesn't include LINEAR or NONLINEAR). We should probably change the label to make that clear.